marimbabot_planning/src/planning.cpp

```diff
@@ -274,8 +306,8 @@ void Planning::action_server_callback(const marimbabot_msgs::HitSequenceGoalCons
 
     try {
         // Set the max velocity and acceleration scaling factors
-        move_group_interface_.setMaxVelocityScalingFactor(0.5);
-        move_group_interface_.setMaxAccelerationScalingFactor(0.5);
+        move_group_interface_.setMaxVelocityScalingFactor(0.95);
```

Comment:
This factor does not multiply with the "original"/true speed of the robot but the restrictions that is was set to in the config file, right?

Reply:
Exactly. If set to `1.0` moveit returns the fastest plan for the given planner and constraints. You can lower the speed of the robot by decreasing it. But if is different to the robots velocity and acceleration limits which are statically defined. Idk why it was so low tbh.. I think someone might decreased it because the robot failed to follow the trajectory, but this was most likely a different issue where the interpolation returned Nan for trajectories with a length of 1.